### PR TITLE
GEODE-3539: Consolidate CliUtil and DataCommandUtils, DataCommandsUtil

### PR DESCRIPTION
**Pull request by jinmeiliao**
* remove DataCommandsUtil
* move methods from DataCommandUtils to CliUtil if applicable.
* move methods from DataCommandUtils to individual command if only used by one command

This is the work triggered by the effort to refactor GemfireDataCommandsDUnitTest. Thought I should stop here to make the change set small.

### CODE REVIEW

File: geode-core/src/main/java/org/apache/geode/management/internal/cli/commands/ExportDataCommand.java [hunk 1]
```diff
@@ -107,4 +107,24 @@ private String defaultFileName(String dirPath, String regionName) {
     }
     return Optional.empty();
   }
+
+  static Result getFunctionResult(ResultCollector<?, ?> rc, String commandName) {
```
Comment (jdeppe-pivotal) [inline geode-core/src/main/java/org/apache/geode/management/internal/cli/commands/ExportDataCommand.java]: Can this also be moved to CliUtil?

File: geode-core/src/test/java/org/apache/geode/management/internal/cli/commands/GemfireDataCommandsDUnitTest.java [hunk 2]
```diff
@@ -589,14 +588,14 @@ public void doTestBug48013() {
...
-    doTestSelectBeansAsResult();
-    doTestSelectBeansWithNestedFieldAsResult();
+    // doTestSelectWithGfshEnvVariables(true);
+    // doTestSelectWithGfshEnvVariables(false);
+    // doTestSelectProjection();
+    // doTestBug48013();
+    // doTestSelectProjectionProcessCommand();
+    // doTestSelectProjectionWithNestedField();
+    // doTestSelectBeansAsResult();
+    // doTestSelectBeansWithNestedFieldAsResult();
```
Comment (jdeppe-pivotal) [inline geode-core/src/test/java/org/apache/geode/management/internal/cli/commands/GemfireDataCommandsDUnitTest.java]: Can these be deleted? If they need to stay, could you add a comment as to why they're commented out.

File: geode-core/src/main/java/org/apache/geode/management/internal/cli/commands/ExportDataCommand.java (hunk 1 above)
Comment (jinmeiliao) [reply geode-core/src/main/java/org/apache/geode/management/internal/cli/commands/ExportDataCommand.java]: moved.

File: geode-core/src/test/java/org/apache/geode/management/internal/cli/commands/GemfireDataCommandsDUnitTest.java (hunk 2 above)
Comment (jinmeiliao) [reply geode-core/src/test/java/org/apache/geode/management/internal/cli/commands/GemfireDataCommandsDUnitTest.java]: this is an accidental change that needs to be reverted.

**Pull request merged**